nebula_hw_interfaces/include/nebula_hw_interfaces/nebula_hw_interfaces_continental/continental_srr520_hw_interface.hpp

```diff
@@ -0,0 +1,147 @@
+// Copyright 2024 TIER IV, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef NEBULA_CONTINENTAL_SRR520_HW_INTERFACE_H
+#define NEBULA_CONTINENTAL_SRR520_HW_INTERFACE_H
+
+#include <nebula_common/continental/continental_srr520.hpp>
+#include <nebula_hw_interfaces/nebula_hw_interfaces_common/nebula_hw_interface_base.hpp>
```

Comment:
`#include ""`

Reply:
Addressed in [b379c81](https://github.com/tier4/nebula/pull/124/commits/b379c81ede85d04262b7cc8c12f674d95febbce5)

Reply:
Addressed in [259b476](https://github.com/tier4/nebula/pull/124/commits/259b4764f48ab8c3aa9a5fb42a724985322c396f)